mcs/class/corlib/Test/System/TypedReferenceTest.cs

```diff
@@ -34,6 +34,7 @@ namespace MonoTests.System
 {
 #if !MONODROID // Type load segfaults the runtime on ARM64 (https://gist.github.com/grendello/334d06c45376602a9afc)
 	[TestFixture]
+	[Category("NotWasm")]
```

Comment:
maybe add `// currently causes the runtime to abort`? (I can't find the "Suggested change" github function)

I will do the merge right after, no need to wait for all the checks

Reply:
added the comment